### DISCUSSION TITLE
Kickoff of Ambassador program

### DISCUSSION THREAD

**Discussion by mhdawson** [nodejs/help#4668] [Announcements]
We've just created the respository for the Ambassador program as outlined in - https://github.com/nodejs/node/blob/main/doc/contributing/advocacy-ambasador-program.md.

We need the help with the following:

* Nominations for the initial ambassadors is open
* PR's to add messages we believe should be promoted
* PRs to add topics where additional content would benefit the community.

**Comment by avivkeller** [comment]
*(For reference)*

https://github.com/nodejs/nodejs-ambassadors
<sup>🔒 Members-Only</sup>
